consensus/types/src/beacon_block_body.rs

```diff
@@ -47,11 +54,12 @@ pub const BLOB_KZG_COMMITMENTS_INDEX: usize = 11;
     cast_error(ty = "Error", expr = "Error::IncorrectStateVariant"),
     partial_getter_error(ty = "Error", expr = "Error::IncorrectStateVariant")
 )]
-#[derive(Debug, Clone, Serialize, Deserialize, Derivative, arbitrary::Arbitrary)]
+#[derive(Debug, Clone, Serialize, Deserialize, Derivative, TreeHash, arbitrary::Arbitrary)]
```

Comment:
I think this `TreeHash` derive here may be unnecessary

Reply:
removed in https://github.com/sigp/lighthouse/pull/4946/commits/d314d12a0f23b65df1d953e18778fdf59d327bfb